src/components/Videos.vue

```diff
@@ -67,6 +78,12 @@ Vue.use(VuePlyr)
 export default {
 	name: 'Videos',
 
+	data: () => ({
+		// This is required so that tracks is declared and reactive
+		// Otherwise updates may fail to make it to plyr
+		tracks: [],
+	}),
+
```

Comment:
```suggestion
	data() {
		// This is required so that tracks is declared and reactive
		// Otherwise updates may fail to make it to plyr
		tracks: [],
	},

```

Reply:
Thanks for the suggestion.
Had to adjust it a little : make data() return the object { tracks: [], }

Reply:
Ah yes, sorry, I didn't wrap it up again :)